ralph/skills/caretake/modes/debug.md

```
@@ -1,194 +1,36 @@
-
-```
-DEBUG SKIPPED no-errors-in-window
-```
-
-and STOP.
+## Behavior
 
-Otherwise pretty-print the **top 5 by `count`**, one block per signature:
+Emit exactly:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)